### ISSUE TITLE
Loss function is poorly suited to the prediction goal

### ISSUE THREAD

**Issue by OliverJMG** [OliverJMG/splid-comp#1]
NLL-Loss function currently used in the training loop evaluates performance on the segment classifications rather than the changes points, so does not reflect the actual goal of the model.

**Comment by OliverJMG** [comment]
New LSTM-based model uses a two-stage loss function that encourages more consistent predictions. Class weights need tweaking but the loss function is no longer a major bottleneck on performance.
